src/cljs/main/broadfcui/common/gcs_file_preview.cljs

```diff
@@ -55,6 +55,7 @@
                          [:span {:style {:marginLeft "1em"}}
                           [:a {:href (common/gcs-object->download-url (:bucket-name props) (:object props))
                                :onClick #(utils/set-access-token-cookie (utils/get-access-token))
+                               :onContextMenu #(utils/set-access-token-cookie (utils/get-access-token))
```

Comment:
Yeah, I would definitely prefer deduping that bit of code. Maybe it should be a util function?
```clojure
(defn refresh-access-token [] (set-access-token-cookie (get-access-token)))
```
in utils.cljs, then just `:onClick utils/refresh-access-token :onContextMenu utils/refresh-access-token` here.